### PR TITLE
AER-2978 Validation on building radius

### PR DESCRIPTION
**Pull request by BertScholten**
This change also means validation on import gets a bit stricter, and OPS characteristics based applications won't allow circular buildings anymore (which they shouldn't anyway).

### CODE REVIEW

File: source/imaer-util/src/main/java/nl/overheid/aerius/validation/BuildingValidator.java [hunk 1]
```diff
@@ -31,42 +33,61 @@ public final class BuildingValidator {
...
 
-  private static void checkBuildingGeometry(final List<BuildingFeature> buildings, final List<AeriusException> errors) {
+  public static void validateBuildings(final List<BuildingFeature> buildings, final BuildingLimits buildingLimits,
+      final List<AeriusException> errors, final List<AeriusException> warnings) {
     for (final BuildingFeature feature : buildings) {
-      // Only polygon and point geometries are supported.
-      if (!(feature.getGeometry() instanceof Polygon || feature.getGeometry() instanceof Point)) {
-        errors.add(new AeriusException(ImaerExceptionReason.GML_GEOMETRY_NOT_PERMITTED, feature.getProperties().getLabel()));
+      final boolean valid = checkBuildingGeometry(feature, buildingLimits, errors);
+      // If geometry is not valid, don't bother checking the rest
```
[the remark this answers is not in the document]
Comment (BertScholten) [reply source/imaer-util/src/main/java/nl/overheid/aerius/validation/BuildingValidator.java]: Good point. Removed this initial validity check all together. This does mean you can get an error that your point-based building is not allowed, and that the radius that you perhaps didn't supply (in which case it'll be 0) is also incorrect, even though that's fine when you change to a polygon-based building. Not sure if that's enough of a reason to add that part again, wdyt?

File: source/imaer-util/src/main/java/nl/overheid/aerius/validation/BuildingValidator.java [hunk 2]
```diff
@@ -31,42 +33,58 @@ public final class BuildingValidator {
...
+    }
+  }
+
+  private static void checkBuildingDiameter(final BuildingFeature feature, final BuildingLimits buildingLimits,
+      final List<AeriusException> errors) {
+    final Building building = feature.getProperties();
+    // When the geometry is a point, that indicates that a circular building is defined.
+    // A circular building consists of a point and a positive diameter.
+    final double diameter = building.getDiameter();
+    if (feature.getGeometry() instanceof Point
```
Comment (Hilbrand) [inline source/imaer-util/src/main/java/nl/overheid/aerius/validation/BuildingValidator.java:85]: I would suggest to also add the support for circular building here
```suggestion
    if (buildingLimits.isCircularBuildingSupported() && feature.getGeometry() instanceof Point
```

File: source/imaer-util/src/main/java/nl/overheid/aerius/validation/BuildingValidator.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hilbrand) [reply source/imaer-util/src/main/java/nl/overheid/aerius/validation/BuildingValidator.java]: See comment inline.

**Review by Hilbrand — APPROVED** [review]
LGTM

**Pull request merged**